File: problemas/iniciante/2235/2235.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    const [a, b, c] = input.trim().split(' ').map(Number);

    if (a === b || a === c || b === c) {
        console.log("S");
    } else if (a + b === c || a + c === b || b + c === a) {
        console.log("S");
    } else {
        console.log("N");
    }

});

File: problemas/iniciante/2235/2235.py
def main():
    A, B, C= map(int, input().split())

    if (A + B == C) or (B + C == A) or (A + C == B) or (A == B or A == C or B == C):
        print("S")
    else:
        print("N")

if __name__ == "__main__":
    main()
